react/ActionMenu/index.jsx

```diff
@@ -167,15 +167,29 @@ const ActionMenuHeader = ({ children }) => {
   return <div className={styles['c-actionmenu-header']}>{children}</div>
 }
 
-const ActionMenuItem = ({ left, children, right }) => {
+const ActionMenuItem = ({ left, children, right, onClick }) => {
+  const onClickEnhanced = e => {
+    if (onClick) {
+      // we need to stop propagation here so that the menu doesn't close itself
+      e.stopPropagation()
+      onClick()
+    }
+  }
+
   return (
-    <Media className={styles['c-actionmenu-item']}>
+    <Media className={styles['c-actionmenu-item']} onClick={onClickEnhanced}>
       {left && <Img className="u-mh-1">{left}</Img>}
       <Bd className={left ? 'u-mr-1' : 'u-mh-1'}>{children}</Bd>
       {right && <Img className="u-mr-1">{right}</Img>}
     </Media>
   )
 }
 
+ActionMenuItem.propTypes = {
+  left: PropTypes.bool,
+  right: PropTypes.bool,
```

Comment:
`left` and `right` seem to be `PropTypes.node` since they are rendered (see https://github.com/cozy/cozy-ui/pull/1174/files#diff-38b8cc61b38494873608fee164497ad7R181 and https://github.com/cozy/cozy-ui/pull/1174/files#diff-38b8cc61b38494873608fee164497ad7R183)

Reply:
Oh right! Thanks. Fixed.